crates/pop-parachains/src/build.rs

```diff
@@ -196,6 +200,36 @@ fn replace_para_id(chain_spec: PathBuf, para_id: u32, generated_para_id: u32) ->
 	Ok(())
 }
 
+/// Replaces the generated relay spec name with the given one in an existing chain spec.
+pub fn replace_relay_spec(
```

Comment:
This approach might be much simpler using serde_json::Value, especially if we are able to read in an existing file to amend in place.

Reply:
This can be resolved